src/Symfony/Component/ErrorHandler/Tests/Command/ErrorDumpCommandTest.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ErrorHandler\Tests\Command;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\TwigBundle\Tests\TestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\ErrorHandler\Command\ErrorDumpCommand;
+use Symfony\Component\ErrorHandler\ErrorRenderer\ErrorRendererInterface;
+use Symfony\Component\ErrorHandler\Exception\FlattenException;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\WebpackEncoreBundle\Asset\EntrypointLookupInterface;
+
+class ErrorDumpCommandTest extends TestCase
+{
+    private string $tmpDir = '';
+
+    public function testDumpPages()
+    {
+        mkdir($this->tmpDir);
```

Comment:
since you already use the FS component in this class, let's us it also for this call